src/main/resources/assets/computercraft/lua/rom/programs/http/pastebin.lua

```diff
@@ -121,40 +50,36 @@ elseif sCommand == "get" then
         return
     end
 
+    print( "Connecting to pastebin.com... " )
```

Comment:
Well, this is categorically a lie. This should at least be after the "file already exists" check.